orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/PurpleData.java

```diff
@@ -44,6 +44,12 @@ public interface PurpleData
     @Nullable
     List<StructuralVariant> allGermlineStructuralVariants();
 
+    @NotNull
+    List<StructuralVariant> allInferredSomaticStructuralVariants();
```

Comment:
It is not clear from the datamodel that the inferred somatic structural variants are not included in `allSomaticStructuralVariants`, this is somewhat confusing! 

I think the clean solution would be to load both PASS and INFERRED sv's into the `all` variable and then downstream users have to filter, but might be more convenient to just rename the `allSomatic` variables to `allPassSomatic`?